chpl/chpl-api/src/main/resources/log4j2-xinclude-file-appenders-json.xml

```diff
@@ -979,4 +979,24 @@
             </Delete>
         </DefaultRolloverStrategy>
     </RollingFile>
+    <RollingFile name="sendEmailJobJson"
+        fileName="${logDir}/scheduler/sendEmailJob.json"
+        filePattern="${logDir}/scheduler/history/sendEmailJob-%d{yyyy-MM-dd}.json"
+        filePermissions="rw-rw-r--">
+        <JsonLayout compact="true" eventEol="true" properties="true" stacktraceAsString="true">
+            <KeyValuePair key="service" value="sendEmailJob" />
+            <KeyValuePair key="dd.trace_id" value="%X{dd.trace_id}" />
+            <KeyValuePair key="dd.span_id" value="%X{dd.trace_id}" />
```

Comment:
Curious about this line. I assume it was copy/pasted from elsewhere, but should the "dd.span_id" have a value of "dd.span_id" or is it correct that both fields have the value trace_id?